BE/scripts/migrate_strategies.py

```
@@ -0,0 +1,176 @@
+        
+        return strategies
+
+
+async def main():
+    """메인 실행 함수"""
+    print("🚀 Starting thumbnail strategy migration...")
+    print(f"📦 Migrating {len(DEFAULT_STRATEGIES)} default strategies\n")
+    
+    try:
+        # 마이그레이션 실행
+        count = await migrate_strategies()
```

Ruff (0.14.14)

[error] Local variable `count` is assigned to but never used

Remove assignment to unused variable `count`

(F841)